src/libraries/System.Threading.Channels/tests/BoundedChannelTests.cs

```diff
@@ -151,7 +152,7 @@ public void TryWrite_TryRead_Many_DropNewest(int bufferedCapacity)
             Assert.Equal(0, result);
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Why is this disabled for IsThreadingSupported?  What is the error you get without this?

Reply:
(I ask because I don't see anything here that should cause it to actually try to block.)

Reply:
Thanks for the catch, I reran the test again and neither this nor the other two in the first commit hung.